tslint.json

```diff
@@ -9,6 +9,35 @@
       "dev",
       ["@kaizen", "@storybook", "@testing-library", "react", "react-dom"]
     ],
-    "ordered-imports": true
+    "ordered-imports": true,
+    "semicolon": false,
+    "no-submodule-imports": false,
+    "no-var-requires": false,
+    "interface-name": false,
+
+    "interface-over-type-literal": {
```

Comment:
The idea here is to _eventually_ switch a rule to a warning -> fix the warnings across the repo -> switch said rule to an error. Rinse and repeat for all the `"severity": "none"` rules. I've disabled a couple of the rules that came with tslint:latest for these reasons:

```
    "semicolon" // this seems to be a standard we roughly follow. Not my taste, but I won't argue it. 
    "no-submodule-imports" // enabling this throws errors for importing Kaizen components. We're safe to use submodule imports for our stories, so i'm disabling this. 
    "no-var-requires" // We `require` our CSS modules, so this needs to stay disabled
    "interface-name": false, // There doesn't seem to be a standard here, but I personally dislike the prefixing my interfaces with "I" and would prefer to use a different convention (like "ButtonProps") 
```

I didn't want to spend too much time on this knowing Delivery Engineering might be creating a shared linting file to keep all repos consistent. This will help us keep things somewhat reasonable until then.